app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> None:
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> None:
+        if args in cache_dict:
+            print("Getting from cache")
+            return cache_dict[args]
```

Comment:
Move it after if-else

Reply:
You can use else)
See example, what i mean, because you have code duplicate: return cache_dict[args]
https://github.com/mate-academy/py-cache-decorator/blob/master/checklist.md#dont-repeat-yourself